packages/api/src/app/api/frontend-fees.ts

```diff
@@ -0,0 +1,55 @@
+import {z} from 'zod'
+import {createTypeGuardFromSchema, fetcher, Fetcher} from '@yoroi/common'
+import {App} from '@yoroi/types'
+
+export const getFrontendFees =
+  (baseUrl: string, request: Fetcher = fetcher) =>
+  async (): Promise<App.FrontendFeesResponse> => {
+    return request<App.FrontendFeesResponse>({
+      url: `${baseUrl}/frontend-fee`,
+      data: undefined,
+      method: 'GET',
+      headers: {'Content-Type': 'application/json'},
+    }).then((response) => {
+      const parsedResponse = parseFrontendFeesResponse(response)
+      if (!parsedResponse)
+        return Promise.reject(new Error('Invalid frontend fee response'))
+
+      return Promise.resolve(parsedResponse)
+    })
+  }
+
+const SwapAggregatorSchema = z.union([
+  z.literal('muesliswap'),
+  z.literal('dexhunter'),
```

Comment:
@stackchain, related to the `dexhunter` in which case will use it? as is the first time I have seen this mentioned. I thought we would be using only `muesliswap`

Reply:
hey @SorinC6 no worries, it's not for now.